README.md

```diff
@@ -127,7 +127,43 @@ The following optional parameters should be used in the `driver_config` for the
  - `poweron` - Power on the new virtual machine. Default: true
  - `vm_name` - Specify name of virtual machine. Default: `<suite>-<platform>-<random-hexid>`
  - `resource_pool` - Name of the resource pool to use when creating the machine. Will search first pool by default, can use value 'Resources' for none.
- - `clone_type` - Type of clone, will default to "full" to create complete copies of template. Needs a VM as template parameter, if "linked" clone desired.
+ - `clone_type` - Type of clone, will default to "full" to create complete copies of template. Other values: "linked", "instant"
+
+## Clone types
+
+### Clone type: full
+
+This takes a VM or template, copies the whole disk and then boots up the machine. Default mode of operation.
+
+### Clone mode: linked
+
+Instead of a full copy, this will use delta disks to speed up the cloning process and uses much less IO operations. The `template` parameter has to be a VM in this case, as a template is not usable. After creation of the delta disks, the machine is booted up and writes to its delta only.
+
+Depending on the underlying storage system, performance might vary greatly compared to full clones.
+
+### Clone mode: instant
+
+The instant clone feature has been available as VMFork in earlier vSphere versions, but without a proper public API. With version 6.7.0, this became an official feature. Instant clones work by not only using a delta disk like linked clones, but also share memory with the source machine. Due to this property, the new machines are already booted up after cloning.
```

Comment:
```suggestion
 The instant clone feature was as VMFork in earlier vSphere versions, but without a proper public API. With version 6.7.0, **this** became an official feature. Instant clones work by not only using a delta disk like linked clones, but also share memory with the source machine. Due to **this** property, the new machines are already booted up after cloning.
```
Does **this** refer to the API? Or to the way instant clones work? Each use of **this** points in a different direction.